rootfs/usr/share/inputplumber/capability_maps/zone_type1.yaml

```diff
@@ -0,0 +1,145 @@
+    target_event:
+      gamepad:
+        button: Guide
+  - name: Screen Left (Long)
+    source_events:
+      - evdev:
+          event_type: KEY
+          event_code: KEY_F18
+          value_type: button
+    target_event:
+      gamepad:
+        button: Keyboard
+  - name: Screen Right 1
+    source_events:
+      - evdev:
+          event_type: KEY
+          event_code: KEY_F17
+          value_type: button
+    target_event:
+      gamepad:
+        button: QuickAccess
+  - name: Screen Right 2
+    source_events:
+      - evdev:
+          event_type: KEY
+          event_code: KEY_F19
+          value_type: button
+    target_event:
+      gamepad:
+        button: QuickAccess2
+  - name: Left Paddle
+    source_events:
+      - evdev:
+          event_type: KEY
+          event_code: KEY_F14
+          value_type: button
+    target_event:
+      gamepad:
+        button: LeftPaddle1
+  - name: Right Paddle
+    source_events:
+      - evdev:
+          event_type: KEY
+          event_code: KEY_F15
+          value_type: button
+    target_event:
+      gamepad:
+        button: RightPaddle1
+  - name: Left Dial Counter-clockwise
+    source_events:
+      - evdev:
+          event_type: REL
+          event_code: REL_HWHEEL
+          axis_direction: negative
+          value_type: button
+    target_event:
+      keyboard: KeyVolumeDown
+  - name: Left Dial Clockwise
+    source_events:
+      - evdev:
+          event_type: REL
+          event_code: REL_HWHEEL
+          axis_direction: positive
+          value_type: button
+    target_event:
+      keyboard: KeyVolumeDown
+  - name: Right Dial Counter-clockwise
+    source_events:
+      - evdev:
+          event_type: REL
+          event_code: REL_WHEEL
+          axis_direction: negative
+          value_type: button
+    target_event:
+      keyboard: KeyVolumeDown
+  - name: Right Dial Clockwise
+    source_events:
+      - evdev:
+          event_type: REL
+          event_code: REL_WHEEL
+          axis_direction: positive
+          value_type: button
+    target_event:
+      keyboard: KeyVolumeDown
+  - name: Mouse Left
+    source_events:
+      - evdev:
+          event_type: KEY
+          event_code: BTN_LEFT
+          value_type: button
+    target_event:
+      mouse:
+        button: Left
+  - name: Mouse Right
+    source_events:
+      - evdev:
+          event_type: KEY
+          event_code: BTN_RIGHT
+          value_type: button
+    target_event:
+      mouse:
+        button: Right
+  - name: Mouse Middle
+    source_events:
+      - evdev:
+          event_type: KEY
+          event_code: BTN_MIDDLE
+          value_type: button
+    target_event:
+      mouse:
+        button: Middle
+  - name: Left Touchpad
+    source_events:
+      - evdev:
+          event_type: REL
+          event_code: REL_WHEEL # horizontal plus vertical
+          value_type: button
+    target_event:
+      mouse:
+        motion:
+          direction: horizontal
```

Comment:
The target mapping here should be a touchpad, not a mouse.

Reply:
Except these are relative events so it seems to break expectations in code somewhere. They are removed now anyways.